api/api_full.go

```diff
@@ -726,8 +726,33 @@ type FullNode interface {
 	EthAccounts(ctx context.Context) ([]ethtypes.EthAddress, error) //perm:read
 	// EthAddressToFilecoinAddress converts an EthAddress into an f410 Filecoin Address
 	EthAddressToFilecoinAddress(ctx context.Context, ethAddress ethtypes.EthAddress) (address.Address, error) //perm:read
-	// FilecoinAddressToEthAddress converts an f410 or f0 Filecoin Address to an EthAddress
-	FilecoinAddressToEthAddress(ctx context.Context, filecoinAddress address.Address) (ethtypes.EthAddress, error) //perm:read
+
+	// `FilecoinAddressToEthAddress` is the recommended API for converting any Filecoin address to an EthAddress.
+	//
+	// This method supports all Filecoin address types:
+	// - "f0" and "f4" addresses: Converted directly.
+	// - "f1", "f2", and "f3" addresses: First converted to their corresponding "f0" ID address, then to an EthAddress.
+	//
+	// Requirements:
+	// - For "f1", "f2", and "f3" addresses, they must be instantiated on-chain, as "f0" ID addresses are only assigned to actors when they are created on-chain.
```

Comment:
I wonder if we should go into detail about this? I remember being confused about this the first time I encountered it; it's perfectly logical to me now but for the uninitiated they may need to know a bit more about what it means to be on chain